src/sources/docker.rs

```diff
@@ -348,12 +318,31 @@ impl DockerSource {
                     self.containers.insert(id.clone(), self.esb.start(id));
                 }
 
-                // Sort for efficient lookup
-                self.ignore_container_id.sort();
                 self
             })
             .map_err(|error| error!(message="Listing currently running containers, failed",%error))
     }
+
+    /// True if passed check
+    fn self_check<'a>(&self, id: &str, image: impl Into<Option<&'a str>>) -> bool {
```

Comment:
I don't think you need this life time here, also the name of this and the doc don't make that much sense could you clarify what this does by updating the name/docs.

Reply:
Compiler is complaining that it's `missing lifetime specifier`.

Maybe this will be clearer.